subsys/bluetooth/host/conn.c

```diff
@@ -972,6 +976,27 @@ struct bt_conn *get_conn_ready(void)
 	return NULL;
 }
 
+struct bt_conn *get_conn_ready(void)
+{
+	if (dont_have_viewbufs()) {
+		/* We will get scheduled again when the (view) buffers are freed. If you
+		 * hit this a lot, try increasing `CONFIG_BT_CONN_FRAG_COUNT`
+		 */
+		LOG_DBG("no view bufs");
+		return NULL;
+	}
+
+#if defined(CONFIG_BT_ISO)
+	/* check for ISO connections first because ISO is time sensitive */
```

Comment:
```suggestion
	/* check for ISO connections first because ISO can be time sensitive */
```

A minor correction - Not all ISO data carries a timestamp or is time sensitive :)